client/galaxy/scripts/components/WorkflowInvocationReport.vue

```diff
@@ -19,18 +26,47 @@ export default {
     },
     data() {
         return {
-            markdownConfig: {}
+            markdownConfig: {},
+            invocationMarkdown: null,
+            edit: false
         };
     },
     created: function() {
-        const invocationId = this.invocationId;
-        const url = getAppRoot() + `api/workflows/foobar/invocations/${invocationId}/report`;
-        this.ajaxCall(url);
+        this.loadMarkdownConfig();
+        window.addEventListener("keyup", this.keyListener);
+    },
+    destroyed: function() {
+        window.removeEventListener("keyup", this.keyListener);
     },
     methods: {
-        ajaxCall: function(url) {
+        keyListener: function(event) {
+            if (event.keyCode == 27) {
```

Comment:
Can we bind this (more or less hidden feature) to something else please? I don't think this fits the standard expectations for hitting an escape key. `e` , `ctrl`+`e`?